zerver/tests/test_invite.py

```diff
@@ -2354,3 +2354,145 @@ def test_create_multiuse_link_invalid_invite_as_api_call(self) -> None:
             },
         )
         self.assert_json_error(result, "Invalid invite_as")
+
+    def test_multiuse_invite_edits_with_malformed_parameters(self) -> None:
```

Comment:
We usually do not have separate tests for failure and success cases except when there are too many checks. We can wait for more feedback here before changing anything.